### PR TITLE
Print failed test suite logs in CI

### PR DESCRIPTION
**Pull request by SkySkimmer**
in addition to having them as artefacts

### CODE REVIEW

**Review by herbelin — APPROVED** [review]
That was indeed easy!!

File: test-suite/report.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ cp summary.log "$SAVEDIR"/
 rm "$FAILED"
 
 # print info
-if [ -n "$APPVEYOR" ] || [ -n "$PRINT_LOGS" ]; then
+if [ -n "$CI" ] || [ -n "$PRINT_LOGS" ]; then
```
Comment (ejgallego) [inline test-suite/report.sh:24]: I still wonder if we shouldn't do this in general? In `test-suite/dune` we set it unconditionally as indeed I _always_ want to have PRINT_LOGS=1 when developing locally too.

**Review by ejgallego — APPROVED** [review]

File: test-suite/report.sh (hunk 1 above)
Comment (SkySkimmer) [reply test-suite/report.sh:24]: It can get pretty verbose, and locally the logs are easily gotten (or just run `make report PRINT_LOGS=1`)

**Comment by ejgallego** [comment]
We can't test without having the test suite actually fail :) Time will tell then.

@coqbot: merge now

**Pull request merged**